PayItForward.Models/DonationDTO.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace PayItForward.Models
+{
+    using System;
+    using PayItForward.Data.Models;
+
+    public class DonationDTO : BaseModel<Guid>
```

Comment:
The base model is for the database model, why the Donation Dto inherits the base model. Do we need this?

Reply:
Yes, in order to not repeat code and write the Id, CteatedOn ...

Reply:
You can make another base object here only for these purposes and only with properties that you need, as you are using only the id from the base model and not the other properties that are from IAuditInfo and IDeletableEntry